packages/app/src/systems/Transaction/utils/tx/index.ts

```diff
@@ -0,0 +1,2 @@
+export * from './tx';
+export * from './tx.types';
```

Comment:
tx.types ? I haven't seen this syntax before

Reply:
now that we have a folder for tx/utils we can rename this file. good catch